src/app/[locale]/_utils/__tests__/common.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { storageLocaleOps } from './../storage';
+import { addSpacesEvery3Chars, localeFromStorage, isBrowser } from '../common';
+import * as common from '../common';
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+describe('Utilities', () => {
+  describe('addSpacesEvery3Chars', () => {

```

Comment:
Should we improve those descriptions? I mean, can we be more expressive?

Reply:
Sure